Say/ViewController.swift

```diff
@@ -92,10 +94,22 @@ class ViewController: NSViewController {
 
     @IBAction func say(_ sender: NSControl) {
         sender.isEnabled = false
-        Say(text: self.textForSpeech, voice: self.selectedVoice).play(true)
-        sender.isEnabled = true
+        self.sayObj = Say(text: self.textForSpeech, voice: self.selectedVoice)
```

Comment:
Consider to use local variable `say` instead of `self.sayObj`.
Generally in apple environment, we don't like to use abbr like 'Obj'.